rudder-lang/src/generator/cfengine/syntax.rs

```diff
@@ -368,9 +368,17 @@ impl Method {
     pub fn build(mut self) -> Vec<Promise> {
         // EXCEPTION: reunite variable_string_escaped resource parameters that appear to be joined from cfengine side
         if self.resource == "variable" && self.state == "string_escaped" {
-            let merged_resource_parameters = self.parameters.join(".");
-            self.parameters = vec![merged_resource_parameters.clone()];
-            self.report_parameter = merged_resource_parameters;
+            let mut parameters = Vec::new();
+            for p in self.parameters.clone() {
+                let param = p
+                    .strip_prefix("\"")
+                    .and_then(|stripped_p| stripped_p.strip_suffix("\""))
+                    .unwrap(); // parameters are quoted so unwrap will work without issue
```

Comment:
wasnt properly escaped. Not really satisfying way to deal with the issue but it works